src/transformation.py

```diff
@@ -95,8 +94,6 @@ def install_packages(packages, user_option=False):
                 '--force-reinstall',
                 package
             ]
-            if user_option:
-                args.insert(6, '--user')
```

Comment:
It is already present above: https://github.com/keboola/python-transformation-v2/blob/344a441edc55defdb41816682052089a3f236798/src/transformation.py#L92